core/lib/protobuf_config/src/proto/proof_data_handler.proto

```diff
@@ -10,6 +10,4 @@ enum ProtocolVersionLoadingMode {
 message ProofDataHandler {
   optional uint32 http_port = 1; // required; u16
   optional uint32 proof_generation_timeout_in_secs = 2; // required; s
-  optional ProtocolVersionLoadingMode protocol_version_loading_mode = 3; // required
-  optional uint32 fri_protocol_version_id = 4; // required; u16
```

Comment:
@pompon0 is it safe to remove fields from proto repr? (I guess that it's probably OK here since it's not in use, but in general)

Reply:
in general it is not. I haven't set up a CI compatibility check yet (https://linear.app/matterlabs/issue/BFT-446/enable-protobuf-compatibility-ci-for-zksync-era), so it was not detected automatically. You should add `reserved 3,4;` and `reserved "protocol_version_loading_mode","fri_protocol_version_id"` like here https://protobuf.dev/programming-guides/proto3/#fieldreserved. Until it is not in use though, it doesn't really matter.